subekashi/static/subekashi/css/common.css

```
@@ -169,25 +201,34 @@ a {
 
 
 /* メニュー */
-#menuarticle {
+.menuarticle {
     display: none;
-    margin-top: 64px;
+}
+
+.menuarticle.shown {
+    display: block!important;
+    margin-top: 0;
     height: 100vh;
     width: 100vw;
-    top: 0;
     left: 0;
     background: linear-gradient(180deg, #000, #000, 70vh, #005);
 }
 
-#pages p {
-    margin-left: 32px;
-    padding: 10vh 0 0 calc(50% - 550px);
+.menuarticle h2 {
+    margin-left: 32px!important;
+    margin-bottom: 0!important;
+    padding: 0 0 0 calc(50% - 550px);
     font-family: 'Noto Sans JP', sans-serif;
     font-size: 45px !important;
     line-height: 75px !important;
 }
 
-#pages a {
+.menuarticle ul {
+    margin-top: 0px
+}
+
+.menuarticle li {
```

フォントサイズ等はaタグに適応されているので修正が必要です。以下が修正案です。
```suggestion
.menuarticle li a {
```